.dockerignore

```diff
@@ -0,0 +1,3 @@
+**/.git
+**/.node_modules
```

Comment:
Shouldn't this just be `**/node_modules`?

Reply:
*.lock might be good here too?

Reply:
good ideas